common/prometheus-alertmanager-base/values.yaml

```diff
@@ -36,6 +36,9 @@ additionalPeers: []
 # If omitted it is generated automatically, which usually leads to different clusterLabels.
 clusterLabel: ccloud
 
+# OnNamespace or None to match all namespaces.
```

Comment:
```suggestion
# OnNamespace or None to match all namespaces. Default to None here to be independent from the deployed namespace since it would be locked to prometheus-alertmanager otherwise.
```